src/patternfly/components/Table/table-grid.scss

```
@@ -124,7 +124,9 @@
   --#{$table}--m-grid__check--favorite--action--MarginInlineStart: calc(var(--#{$table}--m-grid__check--favorite--MarginInlineStart) + var(--#{$table}--m-grid__favorite--action--MarginInlineStart));
 
   // * Table grid toggle icon
-  --#{$table}__toggle__icon--Transition: .2s ease-in 0s;
+  --#{$table}__toggle-icon--TransitionDuration: var(--pf-t--global--motion--duration--icon--default);
+  --#{$table}__toggle-icon--TransitionTimingFunction: var(--pf-t--global--motion--timing-function--default);
+  --#{$table}__toggle-icon--Transition: transform var(--#{$table}__toggle-icon--TransitionDuration) var(--#{$table}__toggle-icon--TransitionTimingFunction); // TODO remove in breaking change along with shorthand property
```

Needs to retain the old var name so it isn't a breaking change.

```suggestion
  --#{$table}__toggle__icon--Transition: transform var(--#{$table}__toggle-icon--TransitionDuration) var(--#{$table}__toggle-icon--TransitionTimingFunction); // TODO remove in breaking change along with shorthand property
```